prov/efa/src/rdm/rxr_op_entry.c

```diff
@@ -282,6 +307,27 @@ void rxr_tx_entry_try_fill_desc(struct rxr_op_entry *tx_entry,
 	}
 }
 
+int rxr_tx_entry_prepare_to_be_read(struct rxr_op_entry *tx_entry, struct fi_rma_iov *read_iov)
```

Comment:
I prefer rename it to `rxr_tx_entry_prepare_for_read`

Reply:
`prepare_to_be_read` is more precise, `prepare_for_read` is ambiguous because it can mean either `prepare_to_be_read` or `prepare_to_post_read`. Both can happen for a tx_entry.